Adding meal vouchers as payment method (#618)
mollie/api-documentation: source/reference/v2/orders-api/create-order.rst

       .. note:: For selling digitally delivered goods through PayPal, you will need to request PayPal to `enable this on
                 your account <https://developer.paypal.com/docs/classic/express-checkout/digital-goods/IntroducingExpressCheckoutDG/>`_.

   * - ``category``

       .. type:: string
          :required: false

     - The category of product bought. Must be one of the following values:

>        * ``food_and_rink``

Reviewer: ```suggestion
       * ``food_and_drinks``
```

       * ``home_and_garden``

>        * ``gift_and_flowers``

Reviewer: ```suggestion
       * ``gifts_and_flowers``
```

>        .. note:: This parameter is optional. However, *one* of your orderlines should contain it if

Reviewer: ```suggestion
       .. note:: This parameter is optional. However, *one* of your order lines should contain it if
```

       .. note:: For selling digitally delivered goods through PayPal, you will need to request PayPal to `enable this on
                 your account <https://developer.paypal.com/docs/classic/express-checkout/digital-goods/IntroducingExpressCheckoutDG/>`_.

   * - ``category``

       .. type:: string
          :required: false

     - The category of product bought. Must be one of the following values:

>        * ``food_and_drink``

Reviewer: ```suggestion
       * ``food_and_drinks``
```

       * ``home_and_garden``
       * ``gifts_and_flowers``

> 

Reviewer 2: I am wondering if we should have a string identifier `other` here for everything else. It would make it nice. What do you think @MollieRick @ndijkstra @sandervanhooft ?

Author: Just `null` or leave it out?

>        .. note:: This parameter is optional. However, *one* of your order lines should contain it if

Reviewer 2: Can you add this to the examples at the end of the page?
